sink/src/main/kotlin/org/neo4j/connectors/kafka/sink/SinkStrategy.kt

```diff
@@ -89,7 +89,7 @@ data class SinkMessage(val record: SinkRecord) {
             }
       }
       converted
-    }
```

Comment:
what would happen without this change?

Reply:
so the `org.apache.kafka.connect.json.JsonConverter` when configured with `"schemas.enable": false`, does send a schema value of `null` and in that case the previous code block would just return `null`. With this change we pass through the `value` event the `schema` is null.